src/seedu/addressbook/tagging/Tagging.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.tagging;
+
+/**
+ * Represents an interaction between a Person and a Tag
```

Comment:
A more precise description is `Represents an adding or removing a Tag to/from a Person`.